### PR TITLE
Database/World: made the database structure similar to TrinityCore

### PR DESCRIPTION
**Pull request by Helias**
Made the database structure similar to TrinityCore

### CODE REVIEW

File: src/game/Globals/ObjectMgr.cpp [hunk 1]
```diff
@@ -5391,7 +5392,7 @@ void ObjectMgr::LoadNpcTextLocales()
         "Text0_0_loc6, Text0_1_loc6, Text1_0_loc6, Text1_1_loc6, Text2_0_loc6, Text2_1_loc6, Text3_0_loc6, Text3_1_loc1, Text4_0_loc6, Text4_1_loc6, Text5_0_loc6, Text5_1_loc6, Text6_0_loc6, Text6_1_loc6, Text7_0_loc6, Text7_1_loc6, "
         "Text0_0_loc7, Text0_1_loc7, Text1_0_loc7, Text1_1_loc7, Text2_0_loc7, Text2_1_loc7, Text3_0_loc7, Text3_1_loc1, Text4_0_loc7, Text4_1_loc7, Text5_0_loc7, Text5_1_loc7, Text6_0_loc7, Text6_1_loc7, Text7_0_loc7, Text7_1_loc7, "
         "Text0_0_loc8, Text0_1_loc8, Text1_0_loc8, Text1_1_loc8, Text2_0_loc8, Text2_1_loc8, Text3_0_loc8, Text3_1_loc1, Text4_0_loc8, Text4_1_loc8, Text5_0_loc8, Text5_1_loc8, Text6_0_loc8, Text6_1_loc8, Text7_0_loc8, Text7_1_loc8 "
-        " FROM locales_npc_text");
+" FROM locales_npc_text");
```
Comment (ShinDarth) [inline src/game/Globals/ObjectMgr.cpp]: indentation

**Comment by DevRival** [comment]
Nice :+1:

**Comment by Helias** [comment]
Did you test it?

**Comment by ShinDarth** [comment]
@Yehonal what about merge?

**Comment by Yehonal** [comment]
Please if someone can test it before, then i'll merge it

**Comment by Krath** [comment]
Tested it, no problems for me. No problems in my daily run.

**Comment by Bodeguero** [comment]
**@Helias the structure is now equal to the new revision of trinitycore?

Nice Job!**

**Comment by Helias** [comment]
Only a part, not the entire structure

**Pull request merged**